josemak25/hairx-client
# Ft regimen setup

## What this PR does (Screenshots are at the bottom)

Updates Regimen setup screen. 

#### Description of Task to be completed?

Regimen setup screen is to show the user 10 questions. So far, it is showing 3 out of 10 questions.

#### How should this be manually tested?

After cloning the branch:

**a.** run 'yarn install' to install dependencies
**b.** run 'yarn start' to open expo and scan the app image onto your mobile device
**c.** Click on 'start regimen setup' to view changes

javascript
git checkout develop

git pull origin develop

yarn install

yarn test

Using an emulator: Launch the app, and you should see the home screen after loading.

#### **Screenshots**

![IMG_1334](https://user-images.githubusercontent.com/8383707/79900791-b20ba800-8406-11ea-85f8-290c42668bea.PNG)
![IMG_1335](https://user-images.githubusercontent.com/8383707/79900810-ba63e300-8406-11ea-99f0-fa9e0c695ce9.PNG)
![IMG_1336](https://user-images.githubusercontent.com/8383707/79900822-c18af100-8406-11ea-91b5-f8ab82e08176.PNG)
